### ISSUE TITLE
support train groups

### ISSUE THREAD

**Issue by ahicks92** [Factorio-Access/FactorioAccess#272]
What it says.  We need someone sighted to figure out if the sighted UX matches the API and if the API is as simple as it looks, then we need a proper train schedule UI, then we need to plug it into there.

While we could do it without a proper train UI I don't think it's worth it.

**Comment by EphDoering** [comment]
Sighted UX has the ability to select from an existing train group, or type in a new train group. I'm pretty sure it's that simple.

**Comment by ahicks92** [comment]
Existing needs definition: do the groups persist without trains? I think you told me no on this one specifically but the important question is to find out if we need an API request or something to list groups, or if iterating over trains does the trick.

**Comment by EphDoering** [comment]
Iterating trains does the trick as far as I can tell. There may need to be a force filter for PvP, but that's quite the edge case.

**Comment by ahicks92** [comment]
Yes, this issue sort of ended up being a duplicate of the other one wrt groups, but leaving it open anyway until that's more clear.
